app/controllers/application_controller.rb

```diff
@@ -211,4 +211,22 @@ def logged_in_as(roles)
     end
     has_valid_role
   end
+
+  def sort_feed(wikis)
+    if params[:sort] == "title"
+      @sorted_wikis = wikis.sort_by { |wiki| wiki.title.downcase }
+    elsif params[:sort] == "edits"
+      @sorted_wikis = wikis.sort_by { |wiki| wiki.revisions.length }
+      @sorted_wikis.reverse!
+    elsif params[:sort] == "page_views"
+      @sorted_wikis = wikis.sort_by(&:views)
+      @sorted_wikis.reverse!
+    elsif params[:sort] == "likes"
+      @sorted_wikis = wikis.sort_by(&:cached_likes)
+      @sorted_wikis.reverse!
+    else
+      @sorted_wikis = wikis
+    end
+    @sorted_wikis
```

Comment:
You can remove this and it will return the last thing you called. So `@sorted_wikis.reverse`w

Reply:
Hey @sssash18, did you get some time to checkout @Uzay-G review above :point_up: